src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
+        super(color);
+        this.side=side;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
```

Comment:
Ensure that the `super.draw()` call is necessary. This depends on the implementation of the `Figure` class. If `Figure` does not have a `draw()` method or if it does not need to be called, this line can be removed.

Reply:
Agree